cmd/cluster/core/create.go

```diff
@@ -1008,43 +998,3 @@ func parseTolerationString(str string) (*corev1.Toleration, error) {
 
 	return &toleration, nil
 }
-
-// validateMgmtClusterAndNodePoolCPUArchitectures checks if a multi-arch release image or release stream was provided.
-// If none were provided, checks to make sure the NodePool CPU arch and the management cluster CPU arch match; if they
-// do not, the CLI will return an error since the NodePool will fail to complete during runtime.
-func validateMgmtClusterAndNodePoolCPUArchitectures(ctx context.Context, opts *RawCreateOptions, kc kubeclient.Interface, imageMetadataProvider hyperutil.ImageMetadataProvider) error {
-	validMultiArchImage := false
-
-	// Check if the release image is multi-arch
-	if len(opts.ReleaseImage) > 0 && len(opts.PullSecretFile) > 0 {
-		pullSecret, err := os.ReadFile(opts.PullSecretFile)
-		if err != nil {
-			return fmt.Errorf("failed to read pull secret file: %w", err)
-		}
-		validMultiArchImage, err = registryclient.IsMultiArchManifestList(ctx, opts.ReleaseImage, pullSecret, imageMetadataProvider)
-		if err != nil {
-			return err
```

Comment:
Rather than outright removing the multi-arch validation function, we could just log an error message here.